wiremock/stubApis.ts

```diff
@@ -37,6 +37,21 @@ premises.forEach(p => {
       },
     }),
   )
+  stubs.push(async () =>
+    stubFor({
+      request: {
+        method: 'POST',
+        url: `/premises/${p.id}/booking/new`,
+      },
+      response: {
+        status: 200,
+        headers: {
+          'Content-Type': 'application/json;charset=UTF-8',
+        },
+        body: "{id: 'bookingID', {{request.body}}}",
```

Comment:
Would `{id: 'bookingID', ...request.body}` not do the same thing here?

Reply:
to be honest I'm not sure, I was following the [Wiremock docs](https://wiremock.org/docs/response-templating/) for this.
We don't have access to a 'request' variable but I think the double braces syntax allows Wiremock to do some magic behind the scenes.